docs/docs/sources/facebook.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Facebook Source
+sidebar_label: Facebook Source
```

Comment:
The directory is already called `sources`, so this tree will read like

sources
  facebook source
  google source

So I think you can drop

Reply:
must drop :)